### PR TITLE
#1596 making status icon accessible

### LINKED ISSUE #1596: Remove aria-hidden to voice aria-label on icons
Icons showing status need aria-hidden removed if the aria-label is to be voiced on the Assignment Planning view

[MyLA Technical Review 7/24 - row 11](https://docs.google.com/spreadsheets/d/186xZEvxhbh55jLR_mRXT9tMrmFUlcBlUtkN5T7BuT64/edit?gid=886470260#gid=886470260)

### PR DESCRIPTION
**Pull request by pushyamig**
Fixes #1596

if you test this with VO it is not voiced the icon. But I talked with Gonzalo and he checked with NVDA sees to be voicing the label. So he likes the change. (I avoided testing in screen readers since accessibility team don't recommends it)

https://github.com/user-attachments/assets/86eacfe1-7c27-4eb8-b936-e6af0eab4e8b

### CODE REVIEW

File: assets/src/components/AssignmentTable.js [hunk 1]
```diff
@@ -527,7 +527,7 @@ function AssignmentTable (props) {
                     </TableCell>
                     <TableCell className={classes.veryNarrowCell}>
                       <Tooltip title={a.graded ? assignmentStatus.GRADED : a.submitted ? assignmentStatus.SUBMITTED : assignmentStatus.UNSUBMITTED}>
-                        {a.graded ? <GradedIcon className={classes.graded} /> : a.submitted ? <SubmittedIcon className={classes.ungraded} /> : <UnsubmittedIcon className={classes.unsubmitted} />}
+                        {a.graded ? <GradedIcon className={classes.graded} titleAccess='Graded' /> : a.submitted ? <SubmittedIcon className={classes.ungraded} titleAccess='Pending Grade' /> : <UnsubmittedIcon className={classes.unsubmitted}  titleAccess='Unsubmitted'/>}
                       </Tooltip>
```
Comment (pushyamig) [inline assets/src/components/AssignmentTable.js:531]: https://v5.mui.com/material-ui/api/svg-icon/

adding `titleAccess` removes `aria-hidden` attribute

Comment (pushyamig) [reply assets/src/components/AssignmentTable.js:531]: `Before`
Screenshot 2024-08-29 at 12 37 28 PM

`After`
Screenshot 2024-08-29 at 12 33 19 PM

**Comment by pushyamig** [comment]
This PR is ready for review

**Review by jaydonkrooss — APPROVED** [review]
looks good on my local instance

**Pull request merged**